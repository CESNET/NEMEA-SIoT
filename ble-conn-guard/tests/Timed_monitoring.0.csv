time TIMESTAMP,macaddr INCIDENT_DEV_ADDR,uint32 ALERT_CODE,uint32 DURATION,uint8 ATYPE,string CAPTION
2019-09-25T15:24:54.242,6d:73:5a:f3:7b:98,0,3350,0,"The device 6d:73:5a:f3:7b:98 was in use for 3350us."
2019-09-25T22:25:02.323,6d:73:5a:f3:7b:98,0,3477,0,"The device 6d:73:5a:f3:7b:98 was in use for 3477us."
