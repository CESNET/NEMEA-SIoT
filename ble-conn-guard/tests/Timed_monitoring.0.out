2019-09-25T22:25:02.323,6d:73:5a:f3:7b:98,0,3477,0,"The device 6d:73:5a:f3:7b:98 was in use for 3477us."
